notebooks/time-to-merge-prediction/thoth-station/thoth-ttm-deployment-config.yaml

```diff
@@ -0,0 +1,66 @@
+apiVersion: machinelearning.seldon.io/v1
+kind: SeldonDeployment
+metadata:
+  name: ttm-model-thoth
+spec:
+  name: ttm-model-thoth
+  predictors:
+  - componentSpecs:
+    - spec:
+        containers:
+        - image: quay.io/supathak/github-ttm-test:v2
```

Comment:
Hi @oindrillac ,
I believe in future we will be replacing this image to image from aicoe org right? or if it is ok to keep it ?

Reply:
@oindrillac - Should we merge this PR for now, or are we waiting on that issue to be resolved?

Reply:
Yeah that's alright. 🚀